openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/pre-migration.py

```diff
@@ -3,8 +3,33 @@
 from openupgradelib import openupgrade
 
 
+def merge_pos_order_return_module(env):
+    module_pos_order_return = env["ir.module.module"].search(
+        [("name", "=", "pos_order_return")]
```

Comment:
This is not valid, as at this stage the module is already merged in `point_of_sale`. You should inspect traces at DB structure level (if a column or table exists).

Reply:
Is this code then wrong?: https://github.com/oca/openupgrade/blob/15.0/openupgrade_scripts/scripts/website_crm/15.0.2.1/pre-migration.py#L6-L9

Reply:
Not in this case, as you make the changes here (one marked as to remove, and the other to install), not on apriori.py: https://github.com/OCA/OpenUpgrade/blob/15.0/openupgrade_scripts/apriori.py

Reply:
Aah, oks :)

So @stefan-tecnativa hook into our fields to infer we've got the module. You can use `column_exists` https://github.com/OCA/openupgradelib/blob/d0aee908031876a95dc36c7a9a2c847186155aa2/openupgradelib/openupgrade_tools.py#L36

Reply:
@chienandalu is it correct now? I am only checking one field, is in this case mandatory to check both?

Reply:
Yes, you can mark it as resolve and request @pedrobaeza to review again